templates/keys.epp

```diff
@@ -0,0 +1,6 @@
+# THIS FILE IS MANAGED BY PUPPET
+# <%= file %>
```

Comment:
There is another PR open now to remove that to avoid spurious changes. I'll merge that one, and rebase here before the release, but not today.